File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management/Broomstick.cs
namespace WizardManagement {
  public class Broomstick : CleaningImplement {
    private int speed;

    public int Speed
    {
        get { return speed; }
        set { speed = value; }
    }

    public Broomstick(string brand, int speed) : base(brand)
    {
      this.speed = speed;
    }

    public int GetSpeed(){
      return this.speed;
    }

    public string Fly(){
      return "mounting broom, running, skipping, flying!";
    }
  }

}

File: week_06/day_3/composition/Composition_Lesson_End/app/src/main/java/example/codeclan/com/composition_lesson_end/wizard_management/Broomstick.java
package example.codeclan.com.composition_lesson_end.wizard_management;

import example.codeclan.com.composition_lesson_end.behaviours.Flyable;

/**
 * Created by user on 28/08/2017.
 */

public class Broomstick extends CleaningImplement implements Flyable {

    int speed;

    public Broomstick(String brand, int speed){
        super(brand);
        this.speed = speed;
    }

    public int getSpeed(){
        return this.speed;
    }

    public String fly(){
        return "mounting broom, running, skipping, flying!";
    }
}
